### PR TITLE
Switch to mocha/chai/jsdom

### LINKED ISSUE #8: Switch to Mocha/Chai/JSDOM
- Drop karma and phantomjs

### PR DESCRIPTION
**Pull request by c0**
Remove karma and phantomjs for the faster jsdom and mocha directly

Closes #8

### CODE REVIEW

**Comment by colinrymer** [comment]
Woot!

File: karma.conf.js [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-
-    plugins: [
-      'karma-phantomjs2-launcher',
-      'karma-mocha',
-      'karma-chai',
-      'karma-mocha-reporter',
-      require("karma-webpack")
-    ]
-  });
-};
```
Comment (colinrymer) [inline karma.conf.js:41]: :smile:

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "start": "npm install && PORT=3000 NODE_PATH=$NODE_PATH:./src node --harmony bin/server.js",
     "lint": "eslint src",
     "lint:fix": "npm run lint -- --fix",
-    "test": "karma start",
+    "test": "NODE_PATH=./src:./test mocha --compilers js:babel-core/register --timeout 5000 --require ./test/setup.js --recursive src/**/**/__tests__/*.js",
```
Comment (colinrymer) [inline package.json]: This seems like a lot of configuration in the command args. Any way we can pull this into a file, etc.?

Comment (c0) [inline package.json]: Pushed up a change. `test/mocha.opts` FTW http://mochajs.org/#mocha-opts

**Comment by colinrymer** [comment]
:+1:

**Pull request merged**